llvm/lib/Transforms/Instrumentation/IndirectCallPromotion.cpp

```diff
@@ -132,6 +132,14 @@ static cl::opt<int> ICPMaxNumVTableLastCandidate(
     "icp-max-num-vtable-last-candidate", cl::init(1), cl::Hidden,
     cl::desc("The maximum number of vtable for the last candidate."));
 
+static cl::opt<DenseSet<StringRef>> ICPIgnoredBaseTypes(
```

Comment:
cl::list is the typical way of supporting an option that takes a list. Can you just use that instead of adding another cl::opt parser?